mms/benchmarks/benchmark.py

```diff
@@ -184,12 +184,18 @@ def run_single_benchmark(jmx, jmeter_args=dict(), threads=100, out_dir=None):
             else:
                 docker_run_call = "{} run --name {} -p 8080:8080 -v {}:/mxnet-model-server -itd {}".format(docker, container, MMS_BASE, docker_path)
                 run_process(docker_run_call)
-                run_process("{} exec -it {} sh -c 'cd /mxnet-model-server && pip install -U -e .'".format(docker, container), shell=True)
-
-
-        with ChDir(MMS_BASE):
-            run_process('python setup.py bdist_wheel --universal')
-        docker_start_call = "{} exec {} mxnet-model-server --start".format(docker, container)
+                run_process("{} exec -it {} sh -c 'cd /mxnet-model-server && python setup.py bdist_wheel --universal && pip install -U -e .'".format(docker, container), shell=True)
+
+        if pargs.gpus:
+            # Need to update config file to use num gpus....
+            with ChDir(MMS_BASE):
+                config_prop_path = "frontend/server/src/test/resources/config.properties"
+                sed_cmd = "sed -i -e 's/use_ssl/#\ use_ssl/; s/private_key_file/#\ private_key_file/; s/model_server_home=..\/../model_server_home=\/mxnet-model-server\//; s/model_store=..\/modelarchive\/src\/test\/resources\/models/model_store=\/mxnet-model-server\/frontend\/modelarchive\/src\/test\/resources\/models\//; s/certificate_file/#\ ceretificate_file/; s/#\ number_of_gpu=1/\ number_of_gpu=4/' {}".format(config_prop_path)
```

Comment:
We should probably, for the sake of completeness, set the config file on both CPU and GPU runs to either with or without GPU.  In addition, the pargs.gpu[0] should contain the number of GPUs on the machine, so pass that in.  Also consider calling this after parse_args and before the benchmarks since it only has to run once.